Make canned meat recipes use the right amount of meat

brannamm opened the pull request:
#### Summary
```SUMMARY: Bugfixes "Make canned meat recipes use the right amount of meat"``` 
#### Purpose of change
Fixes #32743
#### Describe the solution
The amount of meat and the volume of the container for the canned meat recipes in glass jar and 3L glass jar did not add up to the same value before this change and now they do.

#### Describe alternatives you've considered
Halving the volume of canned meat.

brannamm commented:
Jenkins rebuild

brannamm commented:
Jenkins rebuild

RarkGrames commented:
Might be a better solution to half the volume of chunk of meat (which is the basis of every meat product). Currently 296 g meat has 1 L volume, and the density of meat is [around 1](https://www.sciencedirect.com/science/article/pii/S0023643801907625).

brannamm commented:
> Might be a better solution to half the volume of chunk of meat (which is the basis of every meat product). Currently 296 g meat has 0.5 L volume, and the density of meat is [around 1](https://www.sciencedirect.com/science/article/pii/S0023643801907625).

I agree that seems like the best long term solution but it seems like that will be a big overhaul. My change just makes everything consistent without changing everything. Maybe we should make another issue for changing the volume of meat across the board

RarkGrames commented:
> a big overhaul.
What do you mean? Every cooked meat product (cooked meat, jerky, canned meat rehydrated meat, etc.) inherits its values from raw meat. If you change the volume of chunk of meat, they all will copy the volume from it.

brannamm commented:
I was assuming that changing the base meat volume would result in existing recipes not fitting their containers but if what you listed is all there is then I will just go ahead with that tonight or tomorrow

RarkGrames commented:
After going through the whole list, the only meat dish I've found that gets incorrectly modified is the pickled meat in glass jar. There will be 1/2 portions in a jar instead of 1/1. But that is the same case for pickled offal, which is already like that, so they will have to be addressed separately. I will make a PR fixing them if you do this and it gets merged.

brannamm commented:
Thanks for the analysis RarkGrames, I reverted my original changes, halved the volume of chunk of meat, and fixed pickled meat, pickled offal, canned human recipes.

anothersimulacrum commented on data/json/items/comestibles/carnivore.json:
```diff
@@ -86,7 +86,7 @@
     "description": "Freshly butchered meat.  You could eat it raw, but cooking it is better.",
     "weight": 296,
-    "volume": 2,
+    "volume": 1,
```
```suggestion
    "volume": "250 ml",
```

brannamm commented:
Updated the meat cooling unit test since meat now has less volume thus less thermal conductivity and the test relies on guesstimated times and temperature measurements
